packages/webview_flutter/webview_flutter_wkwebview/example/lib/main.dart

```diff
@@ -605,6 +666,67 @@ class SampleMenu extends StatelessWidget {
       },
     );
   }
+
+  Future<void> _showAlert(BuildContext context, String message) async {
+    return showDialog<void>(
+        context: context,
+        builder: (BuildContext ctx) {
+          return AlertDialog(
+            content: Text(message),
+            actions: <Widget>[
+              TextButton(
+                  onPressed: () {
+                    Navigator.of(ctx).pop();
+                  },
+                  child: const Text('OK'))
+            ],
+          );
+        });
+  }
+
+  Future<bool> _showConfirm(BuildContext context, String message) async {
+    final dynamic result = await showDialog<bool>(
+        context: context,
+        builder: (BuildContext ctx) {
+          return AlertDialog(
+            content: Text(message),
+            actions: <Widget>[
+              TextButton(
+                  onPressed: () {
+                    Navigator.of(ctx).pop(false);
+                  },
+                  child: const Text('Cancel')),
+              TextButton(
+                  onPressed: () {
+                    Navigator.of(ctx).pop(true);
+                  },
+                  child: const Text('OK')),
+            ],
+          );
+        });
+
+    return result.runtimeType == bool && result as bool;
```

Comment:
I think this would be easier to read as `result is bool ? result : false`.

Reply:
@bparrishMines I just remove the result of dynamic type on https://github.com/flutter/packages/pull/5795/commits/aebf9483de36ece51334484119904f8bc92a73c2